lib/slosilo/key.rb

```diff
@@ -121,6 +121,12 @@ def stringify value
         value
       else
         value.to_json
+      end.tap do |string|
+        # Ensure that the string is encoded as raw bytes so that we don't have
+        # encoding errors when smushing it together with the salt.
+        if string.encoding != Encoding::ASCII_8BIT
+          string.force_encoding Encoding::ASCII_8BIT
+        end
```

Comment:
Won't  doing that with a string that's not compatible with UTF-8 raise an exception?  Or are all strings convertible to UTF-8?